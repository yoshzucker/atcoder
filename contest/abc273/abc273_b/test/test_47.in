246000000000000 10
