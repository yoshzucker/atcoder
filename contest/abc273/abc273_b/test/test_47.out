246000000000000
